src/models/utils/index.ts

```diff
@@ -23,3 +26,129 @@ export function isFlagEnabled(Flags: number, checkFlag: number): boolean {
   // eslint-disable-next-line no-bitwise -- Flags require bitwise operations
   return (checkFlag & Flags) === checkFlag;
 }
+
+/**
+ * Sets a transaction's flags to its numeric representation
+ *
+ * @param {Transaction} tx A transaction to set its flags to its numeric representation
+ * @returns {void}
+ */
```

Comment:
This isn't the right TSdocs format

Reply:
I'll be fixing the eslint errors in another commit.